lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -426,6 +445,10 @@ def json_error(raw_response)
       def non_fractional_currency?(currency)
         CURRENCIES_WITHOUT_FRACTIONS.include?(currency.to_s)
       end
+
+      def emv_payment?(payment)
```

Comment:
Though about that, but then I'd have to still ask if `payment.respond_to?(:emv_payment?)` since `payment` isn't guaranteed to be a `CreditCard`.